packages/eas-cli/src/credentials/ios/actions/new/RemoveDistributionCertificate.ts

```diff
@@ -0,0 +1,83 @@
+import { AppleDistributionCertificateFragment } from '../../../../graphql/generated';
+import log from '../../../../log';
+import { confirmAsync } from '../../../../prompts';
+import { Account } from '../../../../user/Account';
+import { Action, CredentialsManager } from '../../../CredentialsManager';
+import { Context } from '../../../context';
+import { selectDistributionCertificateWithDependenciesAsync } from './DistributionCertificateUtils';
+import { RemoveSpecificProvisioningProfile } from './RemoveProvisioningProfile';
+
+export class RemoveDistributionCertificate implements Action {
+  constructor(private account: Account) {}
+
+  async runAsync(manager: CredentialsManager, ctx: Context): Promise<void> {
+    const selected = await selectDistributionCertificateWithDependenciesAsync(ctx, this.account);
+    if (selected) {
+      await manager.runActionAsync(
+        new RemoveSpecificDistributionCertificate(this.account, selected)
+      );
```

Comment:
```suggestion
      await new RemoveSpecificDistributionCertificate(this.account, selected).runAsync(manager, ctx)
```
I left `runActionAsync` in other places because it would break tests, but in new code, it shouldn't be necessary. If you don't depend on any preexisting classes then maybe manager also could be dropped (if you remove `implements Action`)